src/app/pages/TopicPage/Pagination/index.test.jsx

```diff
@@ -28,8 +28,10 @@ describe('Topic Pagination', () => {
     // Both the "next" arrow and the "2" button link to page 2
     expect(document.querySelectorAll('a[href*="page=2"]').length).toBe(2);
 
+    // The "previous" arrow should be hidden when user is on page 1
+    expect(document.querySelectorAll('a[href*="page=1"]').length).toBe(1);
```

Comment:
could these query selectors be `getByText` instead?

Reply:
Don't think so - the link text is just `2` and `2` also appears in the `Page 1 of 2` summary block

```
TestingLibraryElementError: Found multiple elements with the text: 2
```